addon/templates/components/nypr-story/header-content.hbs

```diff
@@ -0,0 +1,130 @@
+<div class="flag nudge">
+    {{#if story.headers.brand.logoImage}}
+    <a class="flag-image inherit" href="{{story.headers.brand.url}}">
+        <img alt="{{story.headers.brand.title}}" class="badge squarebadge" 
+            src="{{image-template story.headers.brand.logoImage.template 50 50 story.headers.brand.logoImage.crop 99}}"
+            srcset="{{image-template story.headers.brand.logoImage.template 100 100 story.headers.brand.logoImage.crop 99}} 2x,
+                {{image-template story.headers.brand.logoImage.template 150 150 story.headers.brand.logoImage.crop 99}} 3x"
+            width="50" height="50">
+        <span class="for-screenreaders">{{{story.headers.brand.title}}}</span>
+    </a>
+    <!-- .flag-image -->
+    {{/if}}
+
+    <div class="flag-body">
+        <div class="media media--rev">
+            <div class="media-image">
+                <div id="edit-story"></div>
+            </div>
+
+            {{#if story.headers.links}}
+            <div class="media-body">
+                <div class="text--small dimmed">Published by</div>
+                {{#each story.headers.links as |link|}}
+                <a class="text--mq-bluewhite"  href="{{link.url}}">
+                    {{{link.title}}}
+                </a> {{#unless (eq link story.headers.links.lastObject)}}<span class="text--small dimmed"> by </span>{{/unless}}
+                {{/each}}
+            </div>
+            {{/if}}
+
+        </div>
+        <!-- .media -->
+    </div>
+    <!-- .flag-body -->
+</div>                                
+<!-- .flag -->
+
+<h1 class="h1">{{{story.title}}}</h1>
+
+<div class="l-split flush">
+
+    {{#if story.audioEventually}}
+        {{#if story.audioAvailable}}
+        <div class="btn-group btn-group--withvr btn-group--storyheader">
+          {{!--   <span
+              data-ember-component="listen-button.embedded"
+              data-ember-args='{ "itemPK": "{{story.id}}", "itemTitle": "{{urlquote story.title}}", "duration": "{{story.audioDurationReadable}}", "playContext": "story-header", "type": "blue-boss" }'>
+            </span> --}}
+             {{#listen-button
+                audio=audio
+                type='blue-boss'
+                playConext="story-header"
+                state=(if (eq pk audio.currentId) audio.playState)
+                itemPK=story.id
+                itemTitle=story.title}}
+                Listen {{story.audioDurationReadable}}
+              {{/listen-button}}
+
+            <span
+              data-ember-component="queue-button.embedded"
+              data-ember-args='{ "itemPK": "{{story.id}}", "itemTitle": "{{url-encode story.title}}", "type": "queue-button--storyheader", "region": "StoryHeader" }'>
+            queue button</span>
+
+            {{#if story.audioShowOptions}}
```

Comment:
I'm planning on removing this from the api response. you can replace this with `{{#if (or story.audioMayDownload story.audioMayEmbed)}}`